src/openrct2-ui/windows/RideList.cpp

```diff
@@ -915,16 +915,28 @@ static Widget _rideListWidgets[] = {
                     break;
                 case INFORMATION_TYPE_EXCITEMENT:
                     SortListByPredicate([](const Ride& thisRide, const Ride& otherRide) -> bool {
+                        if (thisRide.ratings.isNull())
+                            return true;
+                        if (otherRide.ratings.isNull())
+                            return false;
                         return thisRide.ratings.excitement <= otherRide.ratings.excitement;
                     });
                     break;
                 case INFORMATION_TYPE_INTENSITY:
                     SortListByPredicate([](const Ride& thisRide, const Ride& otherRide) -> bool {
+                        if (thisRide.ratings.isNull())
+                            return true;
+                        if (otherRide.ratings.isNull())
+                            return false;
                         return thisRide.ratings.intensity <= otherRide.ratings.intensity;
                     });
                     break;
                 case INFORMATION_TYPE_NAUSEA:
                     SortListByPredicate([](const Ride& thisRide, const Ride& otherRide) -> bool {
+                        if (thisRide.ratings.isNull())
+                            return true;
+                        if (otherRide.ratings.isNull())
+                            return false;
```

Comment:
That's true.

But instead of 0 we should use 0xFFFFu similar to kRideRatingUndefined to sort the untested rides below rides that have 0.00 nausea rating eg. Maze.

so
```c++
#include <openrct2/ride/RideRatings.h>
```
and
```c++
const auto leftValue = thisRide.ratings.isNull() ? kRideRatingUndefined : thisRide.ratings.excitement;
const auto rightValue = otherRide.ratings.isNull() ? kRideRatingUndefined : otherRide.ratings.excitement;
return leftValue <= rightValue;
```
should satisfy the sorting request exactly and follow compare requirements